Calculator.c

```diff
@@ -0,0 +1,219 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define MaxLen 100
+int Error = 0;
+
+int IsSpace(char Character)
+{
+    return Character == ' ' || Character == '\t' || Character == '\n';
+}
+
+void RemoveWhiteSpaces(char *Expression)
+{
+    char *Left = Expression;
+    char *Right = Expression;
+    while (*Right != '\0')
+    {
+        if (!IsSpace(*Right))
```

Comment:
It is built in function

Reply:
IsSpace() here is custom built function here which is different from inbluilt isspace() funtion.